src/components/Filters/Filters.enums.ts

```diff
@@ -0,0 +1,10 @@
+export const InputTypes = {
+  string: 'string',
+  multipleString: 'multipleString',
+  uint: 'uint',
+  int: 'int',
+  enum: 'enum',
+  multipleEnum: 'multipleEnum',
+  date: 'date',
+  timeInterval: 'timeInterval',
```

Comment:
`dateInterval` would be probably better since we probably want to select days and not actual timeframe